database/sql/create_procedures.sql

```diff
@@ -10,7 +10,7 @@ PROCEDURE IMPORT_KATEDRY(p_zkratka IN VARCHAR2, p_nazev IN VARCHAR2, p_fakulta I
 
 END SEM_IMPORT;
 
-
+/
```

Comment:
To tady musí být, když u ostatních to není?

Reply:
U PL/SQL veci by to melo byt potreba